bindings/c/fdb_c.cpp

```diff
@@ -510,11 +510,8 @@ FDBFuture* fdb_transaction_get_range_impl(
 	if (target_bytes == GetRangeLimits::BYTE_LIMIT_UNLIMITED)
 		target_bytes = mode_bytes;
 
-	if(target_bytes > 131072){
-		// limit the possible target bytes space to a number that is large enough
-		// to hold a single row with maximum size(110k), but is also a nice power
-		// of 2, because powers of 2 are nice.
-		target_bytes = 131072;
+	if(target_bytes > iteration_progression[max_iteration-1]){
+		target_bytes = iteration_progression[max_iteration-1];
```

Comment:
I made a comment about reverting this logic in a previous PR, and I'll note here additionally that if you revert to the previous logic then it does exactly this for the iteration mode.

Reply:
I moved this to PR #4415 so we can discuss the issue there, rather than here.